godot-git-plugin/src/git_common.cpp

```diff
@@ -19,10 +19,10 @@ void check_git2_errors(int error, const char *message, const char *extra) {
 
 	if (extra) {
 
-		printf("Git API: %s '%s' [%d]%s%s\n", message, extra, error, lg2spacer, lg2msg);
+		printf("Git : %s '%s' [%d]%s%s\n", message, extra, error, lg2spacer, lg2msg);
```

Comment:
Minor nitpick :p 
This would send out the msg as `Git : msg extra error`, would like this to be reverted to the original format: `Git: msg extra error` (notice the space before `:`)